### ISSUE TITLE
RE task is returning zero as the metrics value

### ISSUE THREAD

**Issue by yamini7595** [smitkiri/ehr-relation-extraction#8]
The RE task is giving this warning, UndefinedMetricWarning: Recall and F-score are ill-defined and being set to 0.0 in labels with no true samples. Use `zero_division` parameter to control this behavior.
This is observed for the test set and not for the evaluation set. 
The NER task works completely fine.

**Comment by smitkiri** [comment]
This error just means that there's one label in the y_true list which is not being predicted even once in the test set. It might just be that the model is not performing that well or the number of samples of that specific label is very low in the test set and the model is not able to pick up those few instances. 

It might also be a systematic error in the code (however, this is unlikely since I did not encounter this error while testing on my dataset).

**Comment by yamini7595** [comment]
Thank you for the response. It worked well when tried on another system.

**Comment by vedants03** [comment]
I am facing the same issue, should I train the model on another system with better hardware to solve this problem? Or how can I increase the size of test_set so that model can pickup samples of missing labels too?

**Comment by smitkiri** [comment]
> I am facing the same issue, should I train the model on another system with better hardware to solve this problem? Or how can I increase the size of test_set so that model can pickup samples of missing labels too?

Does it happen every time you train the model?

**Comment by vedants03** [comment]
> > I am facing the same issue, should I train the model on another system with better hardware to solve this problem? Or how can I increase the size of test_set so that model can pickup samples of missing labels too?
> 
> Does it happen every time you train the model?

Thanks for the reply,
Yes everytime, even when I trained on another machine, maybe there's some problem with the datasets. I downloaded Training Data: Track 2 (Training Data v2), Test Data: Task 2 (Relations) and Test Data: Gold standard test data, from the portal. Am I missing something here? After downloading I just organized the data as mentioned in readme file and used generate_data.py file. I have tried everything to solve this issue but nothing has worked for me. Your help is truly appreciated.
